doc/user_interface.md

```diff
@@ -0,0 +1,616 @@
+# User interface
+
+## General principles
+
+The Visual Query Builder interface breaks down into 2 panels:
+
+- the **transformation pipeline on the left**: this is where you can see the
+  ordered series of transformation steps that are applied to your data. When you
+  need to configure a given step, the left panel switches to the **step edition
+  form**
+
+- the **data table on the right**: this is where you can click on you data to apply
+  transformations. The right panel also includes **widgets** above the table.
+  Transformations are available from the columns headers, from the widgets or
```

Comment:
```suggestion
  Transformations are available from the column headers, from the widgets or
```

Reply:
`button` is quite generic too :)
We used the term widget everywhere now and it seems to me that it is well defined in the doh. Consistency is what matters here I think